lisa/sut_orchestrator/azure/tools.py

```diff
@@ -63,12 +63,16 @@ def _initialize(self, *args: Any, **kwargs: Any) -> None:
 
     def get_version(self) -> str:
         result = self.run("-version")
-        if isinstance(self.node.os, CBLMariner):
-            self._command = "/usr/bin/waagent"
-        else:
-            self._command = "/usr/sbin/waagent"
+
         if result.exit_code != 0:
+            if self.node.tools[Ls].path_exists("/usr/bin/waagent"):
+                self._command = "/usr/bin/waagent"
+            elif self.node.tools[Ls].path_exists("/usr/sbin/waagent"):
+                self._command = "/usr/sbin/waagent"
+            else:
+                raise LisaException("waagent path not found")
```

Comment:
`"waagent is not found in path, /usr/bin and /usr/sbin."`

Reply:
updated